app/visit.js

```diff
@@ -6,31 +6,40 @@
  * @param [options.reverse = false]         - When sorting keys, converts all keys to lowercase so
  *                                            that capitalization doesn't interfere with sort order
  * @param [options.ignoreCase = false]      - When sorting keys, converts all keys to
+ * @param [options.depth = false]           - Depth's level sorting keys on a multidimensional object
  * @returns {*}                             - Object with sorted keys, if old wasn't an object
  *                                            returns whatever was passed
  */
 function visit(old, options) {
   options = options || {};
   var ignoreCase = options.ignoreCase || false;
   var reverse = options.reverse || false;
+  var depth = options.depth || false;
```

Comment:
Can we default to `Infinity` instead? I think that'll make more sense and the row below would then simply be `processing = level <= depth`

Reply:
I thought about that but getting lazy. Sorry! :)